src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/automatic-remove/AutomaticRemoveView.tsx

```diff
@@ -23,9 +23,52 @@ export function AutomaticRemoveView(props: Props) {
 
   const dataBrokerCount = CONST_ONEREP_DATA_BROKER_COUNT;
 
-  const { monthlySubscriptionUrl, yearlySubscriptionUrl, ...fixViewProps } =
-    props;
+  let { monthlySubscriptionUrl, yearlySubscriptionUrl } = props;
+  const { ...fixViewProps } = props;
 
+  // format subscription urls
+  if (monthlySubscriptionUrl) {
```

Comment:
Hm, testing this now it seems to be storing and setting the UTM parameters appropriately, but the subscription URL is now `localhost:6060` instead of the `FXA_SUBSCRIPTIONS_URL` setting I have in `.env`. Did this change or is it overriding it accidentally?

Reply:
Hmm, good catch. Not sure, but looking into this.

Reply:
This only seems to happen within dashboard, the landing page one is fine

Reply:
Fixed in https://github.com/mozilla/blurts-server/pull/4047/commits/1a2ef7ada756fab93e0943acc1018b104ff6a86a, thanks